proto/cometbft/types/v2/events.proto

```
@@ -1,7 +1,7 @@
 syntax = "proto3";
-package cometbft.types.v1;
+package cometbft.types.v2;
```

Buf (1.54.0)

Files with package "cometbft.types.v2" must be within a directory "cometbft/types/v2" relative to root but were in directory "proto/cometbft/types/v2".

(PACKAGE_DIRECTORY_MATCH)